src/imitation/util/multi.py

```diff
@@ -0,0 +1,10 @@
+"""Utilities for distributed experiments."""
+import ray.tune.track
+
+
+def ray_tune_active() -> bool:
+  """Returns True if a Ray Tune track session has been initialized."""
+  try:
+    return ray.tune.track.get_session() is not None
+  except ValueError:
```

Comment:
Consider adding comment to explain when/why this occurs?

Reply:
This method doesn't make sense for the current release of Ray. I'm removing this file.